fiftyone/utils/complex_yolo/model.py

```diff
@@ -1,50 +1,225 @@
 """
-Complex-YOLOv3 model adapted from https://github.com/ghimiredhikura/Complex-YOLOv3.
+Complex-YOLOv3 model adapted from
+https://github.com/ghimiredhikura/Complex-YOLOv3.
 
 | Copyright 2017-2022, Voxel51, Inc.
 | `voxel51.com <https://voxel51.com/>`_
 |
 """
-from collections import OrderedDict
-from typing import Tuple, Union
-
 import math
 import numpy as np
-from shapely.geometry import Polygon
-import open3d as o3d
 
 import fiftyone.core.utils as fou
 
 fou.ensure_torch()
 import torch
-import torch.nn as nn
-import torch.nn.functional as nnF
-from torch.autograd import Variable
+from torch import nn
+import torch.nn.functional as F
 
+sg = fou.lazy_import(
+    "shapely.geometry", callback=lambda: fou.ensure_package("shapely")
+)
 
-class Upsample(nn.Module):
-    def __init__(self, scale_factor, mode="nearest"):
-        super(Upsample, self).__init__()
-        self.scale_factor = scale_factor
-        self.mode = mode
 
-    def forward(self, x):
-        x = nnF.interpolate(x, scale_factor=self.scale_factor, mode=self.mode)
-        return x
+class ComplexYOLOv3(nn.Module):
```

Comment:
nit: I think putting the "important" classes and functions at the top of the module is helpful to readers trying to understand why the module exists and what it does